diff --git a/bc/Screens/MiniGame/KinkyDungeon/KinkyDungeonMistress.d.ts b/bc/Screens/MiniGame/KinkyDungeon/KinkyDungeonMistress.d.ts
index 7d781d4..fc7e0e4 100644
--- a/bc/Screens/MiniGame/KinkyDungeon/KinkyDungeonMistress.d.ts
+++ b/bc/Screens/MiniGame/KinkyDungeon/KinkyDungeonMistress.d.ts
@@ -2,9 +2,9 @@ declare let Ally: string;
 declare let Sub: string;
 declare let Enemy: string;
 declare let Jailer: string;
-declare namespace KDMistress {
-    namespace Adventurer {
-        export const name: string;
-        export { Ally as alignment };
+declare let KDMistress: {
+    Adventurer: {
+        name: string;
+        alignment: string;
     }
 }
